ocaml/bytecomp/bytelink.ml

```diff
@@ -212,6 +214,8 @@ let check_consistency file_name cu =
 
 let extract_crc_interfaces () =
   Consistbl.extract !interfaces crc_interfaces
+  |> List.map (fun (name, crc_with_unit) ->
```

Comment:
The problem with returning an array is that this output goes into a `List.filter` in `Bytepackager`.  I've added the type annotation.

Reply:
I think the annotation should be
`Import_info.t array`; otherwise the
code in `Symtable` will try and
unmarshal an array where a list
was marshalled.

Reply:
Actually I've just replaced the annotation with `Array.to_list` now; ok?

Reply:
Looks good.